src/Elcodi/Component/Menu/Entity/Menu/Node.php

```diff
@@ -290,6 +297,10 @@ public function isActive($currentUrl)
      */
     public function isExpanded($currentUrl)
     {
+        if (!($this->subnodes instanceof Collection)) {
```

Comment:
check the VLD: https://3v4l.org/MJuCR/vld#tabs
the `instanceof` has a higher precedence than `!` and both of them give the same output